docs/content/api/module/retry.md

```diff
@@ -8,7 +8,20 @@ title: Mollitia - API - Module - Retry
 ## Usage
 
 The `Retry` module allows you to **retry** a function when it fails.
-You can configure the interval between attempts, and filter the error to see if you want to retry or not.
+You can configure:
+
+* the interval between attemps
+* if the first retry should be done without delay
+* the default interval between retry
+* the factor to be applied for the retry interval (applicable if mode is LINEAR, EXPONENTIAL or JITTER)
+* the mode to be used for the retry. Mode could be:
+  * CONSTANT: The interval between each retry is always the one configured with the interval option
+  * LINEAR: The interval between each retry grows linearly, based on interval and factor configuration
+  * EXPONENTIAL: The interval between each retry grows exponentially, based on interval and factor configuration
+  * JITTER: The interval between each retry uses a Jitter calculation to distribute the retries randomly and avoid peaks of retries. See [Jitter description here](https://aws.amazon.com/blogs/architecture/exponential-backoff-and-jitter/)
```

Comment:
I this link needed, shouldn't we just reference this internal documentation blow (maybe `#jitter-mode` ?)

Reply:
Yep, agree as the document on this link is no longer aligned with our implementation of jitter mode.
Will set internal link instead